packages/azure-themes/src/azure/IExtendedSemanticColors.ts

```diff
@@ -2,6 +2,35 @@ import { ISemanticColors } from 'office-ui-fabric-react';
 
 export interface IExtendedSemanticColors extends ISemanticColors {
   labelText: string;
+  buttonText: string;
+  buttonBackgroundHoveredDefault: string;
+  buttonBorderColor: string;
+  buttonBorderHovered: string;
+  checkboxBorder: string;
+  checkboxBorderHover: string;
+  checkboxBorderChecked: string;
+  checkmarkColorHover: string;
+  checkboxBorder2: string;
```

Comment:
what is checkboxBorder2 for? if this border is for specific use, can we use more descriptive name instead of postfixing it with 2?